js/tests/karma.conf.js

```diff
@@ -57,14 +57,13 @@ module.exports = (config) => {
     coverageIstanbulReporter: {
       dir: jsCoveragePath,
       reports: ['lcov', 'text-summary'],
-      fixWebpackSourcePaths: true,
       thresholds: {
-        emitWarning: true,
+        emitWarning: false,
         global: {
-          statements: 80,
-          lines: 80,
-          branches: 80,
-          functions: 80
+          statements: 89,
```

Comment:
Are you sure about the new numbers? Maybe we should be less strict?

Reply:
For me it's pretty low we should be at least at 95% on those numbers